exporter_test.go

```diff
@@ -31,6 +31,15 @@ import (
 	main "github.com/RedHatInsights/insights-results-aggregator-exporter"
 )
 
+const (
+	expectedVersionMessage       = "Insights Results Aggregator Cleaner version 1.0"
+	expectedAuthorsMessage       = "Pavel Tisnovsky"
+	expectedCopyrightMessage     = "Red Hat Inc."
+	expectedConfigurationMesage1 = "Driver"
```

Comment:
expectedConfigurationMesage1 should be expectedConfigurationMessage1

Reply:
fixed